reconcile/utils/ocm/labels.py

```diff
@@ -30,6 +33,42 @@ def get_subscription_labels(
             yield subscription_label
 
 
+def add_subscription_labels(
+    ocm_api: OCMBaseClient,
+    cluster: ClusterDetails,
```

Comment:
consider using the `OCMCluster` class here because it is sufficient

Reply:
fixed in [22d92f9](https://github.com/app-sre/qontract-reconcile/pull/3714/commits/22d92f97b52facbec4cf4729c3977c5e8452cf84)